pycds/__init__.py

```diff
@@ -211,6 +211,26 @@ class NativeFlag(Base):
                          name='meta_native_flag_unique'),
     )
 
+class DerivedValue(Base):
+    __tablename__ = 'derived_values'
```

Comment:
I've been wondering about that. I shied away from using "obs" in the name because they strictly aren't observations, they are statistics or derived values. But your point about them being derived from obs_raw is well taken.